fgd/redream.fgd

```diff
@@ -1,7 +1,7 @@
-@include "halflife2.fgd"
+@include "base.fgd"
```

Comment:
Every custom fgd (tf.fgd, halflife2.fgd, dod.fgd, cstrike,fgd) always refer to base.fgd, **not** each other.